### PR TITLE
feat [CX-2453]: Expose location object to create and update My Collection artwork mutations

### PR DESCRIPTION
**Pull request by MrSltun**
Resolves [CX-2453]

## Description
This PR exposes the new location object for my collection artworks to be used to enable the AutoSuggest feature

### Update Eigen
- Update the location field in my collection artwork form to include the AutoSuggest feature
- Update mutation queries to include the new field but reading from the old

[CX-2453]: https://artsyproduct.atlassian.net/browse/CX-2453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/artwork/index.ts [hunk 1]
```diff
@@ -202,7 +203,12 @@ export const ArtworkType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLString,
         resolve: (artwork) => artwork.artwork_location,
         description:
-          'Represents the location of the artwork for "My Collection" artworks',
+          'Represents the location of the artwork for "My Collection" artworks - to be deprecated',
```
Comment (olerichter00) [inline src/schema/v2/artwork/index.ts:206]: You can deprecate the field with `deprecationReason: "blablabla`.

**Review by dariakoko — DISMISSED** [review]
Thank you Sultan 🙌

File: _schemaV2.graphql [hunk 2]
```diff
@@ -1694,6 +1694,7 @@ type Artwork implements Node & Searchable & Sellable {
 
   # Represents the location of the artwork for "My Collection" artworks
   artworkLocation: String
+    @deprecated(reason: "Please use `collectorLocation` instead")
```
Comment (olerichter00) [inline _schemaV2.graphql:1697]: 👍

**Review by olerichter00 — APPROVED** [review]
Great work!

**Pull request merged**